monitor/install_collectd_debian.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+set -e
+
+echo "[*] Bisq Server Monitoring installation script"
+
+##### change paths if necessary for your system
+
+ROOT_USER=root
+ROOT_GROUP=root
```

Comment:
```suggestion
ROOT_GROUP=root
ROOT_HOME=~root
```

Reply:
ROOT_HOME is not used in the script

Reply:
It's used in the entropy seeding I added